architecture/2-UserStories/2.2-ListenForSatelliteBeacon.yml

```diff
@@ -0,0 +1,8 @@
+id: 2.2
+name: Listen for Satellite Beacon
+actor: Ground Station Operator 
+behavior: Open ground station beacon monitor
+rationale: View satellite health data packet
+derivedFrom:
+- "2-ViewSatelliteHealthDataPacket.yml"
```

Comment:
should be prepended w/ `2-UserStories/`, i.e., `2-UserStories/2-ViewSatelliteHealthDataPacket.yml`